### ISSUE TITLE
new-app: create extended test to verify that local images get a docker image ref and not an imagestream

### ISSUE THREAD

**Issue by csrwng** [openshift/origin#5000]
Move this test into an extended test... we cannot rely on docker being available for test-cmd.sh

```bash
# ensure a local-only image gets a docker image(not imagestream) reference created.
tmp=$(mktemp -d)
pushd $tmp
cat <<EOF >> Dockerfile
FROM scratch
EXPOSE 80
EOF
docker build -t test/scratchimage .
popd
rm -rf $tmp
[ "$(oc new-app  test/scratchimage~https://github.com/openshift/ruby-hello-world.git --strategy=docker -o yaml |& tr '\n' ' ' | grep -E "from:\s+kind:\s+DockerImage\s+name:\s+test/scratchimage:latest\s+")" ]
```

**Comment by csrwng** [comment]
@bparees

**Comment by smarterclayton** [comment]
Fixed in #5371
